stories/ag/Listings/MobileMapListing.js

```diff
@@ -81,10 +81,31 @@ const singleFamilyProps = {
 
 export const ExampleMobileMapListing = () => {
   const isActive = boolean('isActive', true)
-  return <MobileMapListing {...props} ctaButtons={ctaButtons} isActive={isActive} />
+  const listing = {
+    ...baseListing,
+    isFavorited: boolean('listing.isFavorited', false),
```

Comment:
Is this not throwing errors with lower case "b"? I get a reference error ("boolean is not defined") in console. Maybe babel converts it. Mind converting it (and line 102) to uppercase "B" anyway?